apps/client/src/components/card/card.tsx

```diff
@@ -23,14 +27,17 @@ export const Card: React.FC<Props> = ({
   if (height) {
     style['height'] = height;
   }
-
+  const formattedDate =
+    toastsDate !== undefined ? toastsDate.toString().substring(0, 10) : null;
```

Comment:
It's not really clear what the meaning of `10` in the substring. I suggest using a constant to clarify that